### DISCUSSION TITLE
Drag and drop always creates a new tab, how to change this? (Linux/GNOME)

### DISCUSSION THREAD

**Discussion by Ian2020** [neovide/neovide#2781] [Q&A] [answered]
Hi,

On my machine (Linux/GNOME) dragging a file onto neovide causes a new tab to be opened. I'd rather it opened in the existing tab as I typically only use one. If I drag a lot of files I end up with a lot of tabs to clear up. Is there anyway to configure this behaviour?

Thanks.

**Comment by falcucci** [comment, marked as answer]
As a *non long-term* fix: you can enable buffer only mode to avoid this behavior.

https://neovide.dev/command-line-reference.html#tabs

or in the config file:

```toml
tabs = false
```

and [here](https://github.com/neovide/neovide/issues/2609) there is a discussion in case you want to get involved.

but I guess this was happening mainly on macOS. maybe @fredizzimo can help you more on it.

**Reply by Ian2020** [reply]
Amazing thank you! That has solved it for me and happy to live without tabs.

**Comment by 9mm** [comment]
Hey @falcucci and @fredizzimo - tthanks! -- this no tabs solution sort of works I suppose but there seems to be some weird stuff.

This fixed it in the sense that the following command opens 1 tab now, instead of 2, when running this command:

```
NEOVIDE_TABS=0 neovide -- -c 'cd ~/Code/chia'
```

But it still looks like there's a problem as if the wrong tab was chosen. I would expect a new [buffer 1] but instead I get this. Most specifically notice in my status bar that it thinks the filename is `:cd /Users/...`. 

<img width="224" alt="image" src="https://github.com/user-attachments/assets/f5d22aa6-69f7-42f5-91aa-8fd6c324c13a">

<img width="528" alt="image" src="https://github.com/user-attachments/assets/781300a2-7722-41d5-8176-2670f5a19c1a">


It DOES look like the above command changed the working directly properly but im super confused why it looks way different then if i open neovide and just run the exact same command in the status line. `:cd ~/Code/chia`. in this case, it looks as i would expect

<img width="391" alt="image" src="https://github.com/user-attachments/assets/7eb2813a-921a-44cc-9cb2-315585918b0e">
